server/src/main/java/org/elasticsearch/action/admin/indices/stats/CommonStats.java

```diff
@@ -246,6 +256,7 @@ public CommonStats(StreamInput in) throws IOException {
         recoveryStats = in.readOptionalWriteable(RecoveryStats::new);
         if (in.getVersion().onOrAfter(Version.V_8_0_0)) {
```

Comment:
Needs to change after back port

Reply:
Maybe not since bulk stats is only 8.0? Probably a question for someone on the features team.